### PR TITLE
Adrv9008-2 IIO update

### PR DESCRIPTION
**Pull request by danmois**
## Pull Request Description

Add IIO example for ADRV9008-2.

## PR Type
- [ ] Bug fix (change that fixes an issue)
- [x] New feature (change that adds new functionality)
- [ ] Breaking change (has dependencies in other repos or will cause CI to fail)

## PR Checklist
- [x] I have followed the [Coding style guidelines](http://analogdevicesinc.github.io/no-OS/drivers_guide.html#coding-style)
- [x] I have performed a self-review of the changes
- [x] I have commented my code, at least hard-to-understand parts
- [x] I have build all projects affected by the changes in this PR
- [x] I have tested in hardware affected projects, at the relevant boards
- [x] I have signed off all commits from this PR
- [x] I have updated the documentation (wiki pages, ReadMe etc), if applies

### CODE REVIEW

File: projects/adrv9009/src/app/headless.c [hunk 1]
```diff
@@ -480,6 +480,8 @@ int main(void)
 
 #ifdef IIO_SUPPORT
 #ifdef ADRV9008_2
+	no_os_mdelay(100);
```
Comment (amiclaus) [inline projects/adrv9009/src/app/headless.c:483]: maybe this delay deserves and inline comment so that we know why de delay is required.

**Review by amiclaus — COMMENTED** [review]
looks good, one small comment.

**Comment by danmois** [comment]
V2

- add comment for delay function before IIO server launch

**Review by amiclaus — DISMISSED** [review]

**Comment by danmois** [comment]
V3

- add delay before IIOD server launch for all devices

**Review by dbogdan — APPROVED** [review]

**Review by amiclaus — APPROVED** [review]

**Pull request merged**